spec/features/email_spec.rb

```diff
@@ -30,5 +30,13 @@
     expect(current_path).to eq("/obituaries/#{obit.id}")
     expect(page).to have_content("You Have Successfully Shared this Obituary in Email")
   end 
+
+  it 'Cannot send email if not registered user' do 
```

Comment:
Noticed that I hadnt sad pathed this previously.

Reply:
Thanks for remembering to add a sad path test!

Reply:
yes, definitely next level testing!